artifacts/rhcos/prerelease/rhcos.go

```diff
@@ -0,0 +1,81 @@
+package prerelease
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/containers/image/v5/pkg/compression/types"
+	"kubevirt.io/containerdisks/pkg/api"
+	"kubevirt.io/containerdisks/pkg/docs"
+	"kubevirt.io/containerdisks/pkg/hashsum"
+	"kubevirt.io/containerdisks/pkg/http"
+)
+
+type rhcos struct {
+	Version     string
+	Variant     string
+	getter      http.Getter
+	Arch        string
+	Compression string
+}
+
+var description string = `RHCOS prerelease images for KubeVirt.
+<br />
+<br />
+Visit [https://docs.openshift.com/container-platform/latest/architecture/architecture-rhcos.html) to learn more about Red Hat Enterprise Linux CoreOS.`
+
+func (r *rhcos) Metadata() *api.Metadata {
+	return &api.Metadata{
+		Name:                    "rhcos-dev",
```

Comment:
I am confused about the usage of the words "rhcos-dev" and "pre-release" in this context. Maybe the consistency would be better, if the Name would be "pre-release"?

Reply:
This is the name of the containerdisk in the registry, to be consistent I would name it `rhcos-pre-release`.

Reply:
Switched to `rhcos-pre-release`.